tests/integration/test_microk8s.py

```diff
@@ -34,6 +36,33 @@ async def test_microk8s_cluster(e: OpsTest, series: str, cp_units: int, worker_u
         )
     )
 
+    await e.model.wait_for_idle([application_name], timeout=60 * 60)
+
+    u: Unit = e.model.applications[application_name].units[0]
+
+    # When rbac is not enabled, we can't query for `system:node` cluster    role
+    rc, stdout, stderr = await run_unit(u, "microk8s kubectl get clusterrole system:node")
+    LOG.info("RBAC output %s", (rc, stdout, stderr))
+    assert rc == 1
+
+    await e.model.wait_for_idle([application_name], timeout=60 * 60)
+
+    # When rbac is enabled via configs, we can get `system:node` clusterrole successfully
+    await e.model.applications[application_name].set_config(
+        config={"rbac": "true"},
+    )
+
+    await e.model.wait_for_idle([application_name], timeout=60 * 60)
+
+    rc, stdout, stderr = await run_unit(u, "microk8s status -w")
+    LOG.info("MicroK8s status output %s", (rc, stdout, stderr))
```

Comment:
Please do not unnecessarily log everything, as this will make parsing the logs in case of failures in the future hard.

Keep the logs short and to the point (e.g. if something failed, then print information that would help with understanding _what_ went wrong).

My suggestion would be to only log these lines in case some check has failed.